### PR TITLE
[openwebnet] reduced log messages during UPnP bridge discovery

### LINKED ISSUE #11701: [openwebnet] reduce INFO log messages for upnp discovery
Logged messages while discovering new devices in the network via UPNP should be reduced to just those related to BTicino devices, all other messages should be logged at DEBUG level.

### PR DESCRIPTION
**Pull request by mvalla**
Closes #11701 by changing 2 log messages to DEBUG level.

### CODE REVIEW

File: bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/discovery/BusGatewayUpnpDiscovery.java [hunk 1]
```diff
@@ -226,7 +226,7 @@ private DeviceInfo(RemoteDevice device) {
                     }
                 }
             }
-            logger.info("Found BTicino device: not a OpenWebNet gateway or is not supported (UDN={})", idString);
+            logger.info("Found BTicino device: not a OpenWebNet gateway or not supported (UDN={})", idString);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/discovery/BusGatewayUpnpDiscovery.java:229]: Even this one could be rather DEBUG? 
We should have the minimum of INFO logs.

Comment (mvalla) [reply bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/discovery/BusGatewayUpnpDiscovery.java:229]: I would find useful to tell the user via INFO message that the BTicino device that was found is NOT a gateway that can be used by this binding for integration. Therefore I would leave the level to INFO.

Comment (lolodomo) [reply bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/discovery/BusGatewayUpnpDiscovery.java:229]: I let another maintainer decide.
@kaikreuzer ?

**Comment by lolodomo** [comment]
@fwolter : please have a look, if this is ok for you to leave an INFO level log, you can merge.

**Review by fwolter — APPROVED** [review]

**Pull request merged**